scripts/requirements-doc.txt

```diff
@@ -6,3 +6,6 @@ sphinx>=3.3.0,<3.4.0
 sphinx_rtd_theme
 sphinx-tabs
 sphinxcontrib-svg2pdfconverter
+
+# Used by zephyr_module
```

Comment:
I would put this in scripts/requirements-base.txt and remove it from scripts/requirements-build-test.txt

Reply:
I see your point but this is raising another issue (that was cowardly trying to avoid): building the docs requires almost nothing from `requirements-base.txt`: PyYAML and that's all.

Reply:
the number of people who would want to build docs and but nothing else?

Reply:
Me when I'm using my slow macbook + all the doc writers?

There was a time when people writing code and people writing doc were totally different. I don't miss it but it's not completely gone.

Reply:
well, the point here is that zephyr_module is used everywhere, not only for the docs, so that requirement should be in the base IMO, you can duplicate it in the doc file, but it is needed always.

Reply:
ok, this is pulled by west, so I guess we are fine.

Reply:
> I would put this (pykwalify) in scripts/requirements-base.txt and remove it from scripts/requirements-build-test.txt

Submitted in #31238